tests/conftest.py

```diff
@@ -761,6 +761,7 @@ def factory(
         helpers.wait_for_resource_state(
             dc_pod_obj, constants.STATUS_RUNNING, timeout=180
         )
+        dc_pod_obj.pvc = pvc
```

Comment:
can you try reloading the dc_pod object instead and fetch the pvc information from updated object?

Reply:
I tried using dc_pod_obj.reload() function it didn't work me. 
Some how it is returning none, might be I made mistake:(